lib/linux_admin/security/sshd_config.rb

```diff
@@ -0,0 +1,31 @@
+module LinuxAdmin
+  class Security
+    class SshdConfig
+      require 'linux_admin/service'
+      include Security::Common
+      CONF_FILE = "/etc/ssh/sshd_config"
+
+      SCAP_SETTINGS = {
+        "PermitUserEnvironment" => "no",
+        "PermitEmptyPasswords"  => "no",
+        "Ciphers"               => "aes128-ctr,aes192-ctr,aes256-ctr,aes128-cbc,3des-cbc,aes192-cbc,aes256-cbc",
+        "ClientAliveInterval"   => "900",
+        "ClientAliveCountMax"   => "0"
+      }
+
+      def apply_scap_settings(filename = CONF_FILE)
+        config_text = File.read(filename)
+        SCAP_SETTINGS.each do |k, v|
+          new_line = "#{k} #{v}\n"
+          config_text = replace_config_line(new_line, /^#*#{k}.*\n/, config_text)
+        end
+        File.write(filename, config_text)
+      end
+
+      def restart_service
+        serv = LinuxAdmin::Service.new("sshd")
+        serv.running? ? serv.restart : serv.start
```

Comment:
Just did a quick test and it looks like restart will succeed even if the service is stopped.  So I can just take the method out and call to LinuxAdmin::Service directly.